internal/pkg/manager/spod/spod_controller.go

```diff
@@ -472,6 +472,11 @@ func (r *ReconcileSPOd) getConfiguredSPOd(
 		}
 	}
 
+	// Overwrite the SPOD's default resoruce requirements
+	if cfg.Spec.DaemonResourceRequirements != nil {
+		templateSpec.Containers[bindata.ContainerIDDaemon].Resources = *cfg.Spec.DaemonResourceRequirements
+	}
```

Comment:
I think we could extend this to the `initContainers` as well. wdyt?

Reply:
Do you mean the same resources?

Reply:
I think the two init containers don't have really performance issues, or need more resources. I would let them as they are for now.

Reply:
The idea was mostly to provide users a way to set resource requests/limits across all containers we expose. But I agree that our current init containers are less resource intensive, so happy to not expand this for the time being. :+1: